build.gradle

```diff
@@ -57,7 +80,7 @@ gitHooks {
 jar {
     manifest {
         attributes(
-                "Main-Class": "com.kotlindiscord.database.MainKt"
+                "Main-Class": "com.kotlindiscord.database.migrator.MigratorKt"
```

Comment:
Are you sure the application plugin and JAR should point to different main classes?